README.md

```diff
@@ -47,7 +48,7 @@ docker-compose exec web /bin/sh -c "bundle exec rails db:setup"
 
 Then open http://localhost:3001 to see the app.
 
-## Running specs, linter(without auto correct) and annotate models and serializers
+## Running specs, linter (without auto correct) and annotate models and serializers
```

Comment:
I think its with auto correct

@dankmitchell for input

Reply:
It's more legible and correct I believe.

Reply:
opps I meant this bit

linter (without auto correct)
vs
linter (with auto correct)

not the spacing linter(with auto correct)